### ISSUE TITLE
Theme property not showing dark mode

### ISSUE THREAD

**Issue by radekzz** [henninghall/react-native-date-picker#574]
Version 4.2.5
Text color is changing dynamically according to isDarkMode (**Appearance.getColorScheme()**), but modal background is always white. 
I would appreciate at least an option to set manually modal bgColor so i can use same condition there.

```
<DatePicker
            theme="dark"
            textColor={isDarkMode ? theme.color.white : theme.color.black}
          />
```
<img width="412" alt="dark mode" src="https://user-images.githubusercontent.com/13495893/196970827-6c9ab1bd-fb38-43dd-988f-7de0fff965d5.png">

**Comment by JakeMotta** [comment]
Same issue +1

**Comment by dprajapati1179** [comment]
Same issue I have been facing, Do any workarounds?

**Comment by babyrusa** [comment]
I have this issue as well

**Comment by babyrusa** [comment]
My temporary work around is manually setting "textColor" prop to "black" or a dark color when it's dark mode

**Comment by henninghall** [comment]
Thanks for reporting this, it is now ixed and released in version `4.2.8`
